CHANGELOG.md

```diff
@@ -72,3 +90,5 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.0.2]: https://github.com/zalando-incubator/transformer/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/zalando-incubator/transformer/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/zalando-incubator/transformer/compare/f842c4163e037dc345eaf1992187f58126b7d909...v1.0.0
+
+[@xinke2411]: https://github.com/xinke2411
```

Comment:
What's the purpose of tagging this user here?

Reply:
They are mentioned in a "thank you sentence" [above](https://github.com/zalando-incubator/Transformer/pull/33/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR22), and Codacy was making a big deal about reference links not being at the bottom of the file.